winbuild/MakefileBuild.vc

```diff
@@ -114,11 +114,19 @@ LFLAGS         = $(LFLAGS) "/LIBPATH:$(DEVEL_LIB)"
 
 
 !IF "$(WITH_SSL)"=="dll"
+!IF EXISTS("$(DEVEL_LIB)\libssl.lib")
+SSL_LIBS     = libssl.lib libcrypto.lib
+!ELSE
 SSL_LIBS     = libeay32.lib ssleay32.lib
+!ENDIF
 USE_SSL      = true
 SSL          = dll
 !ELSEIF "$(WITH_SSL)"=="static"
+!IF EXISTS("$(DEVEL_LIB)\libssl.lib")
+SSL_LIBS     = libssl.lib libcrypto.lib ws2_32.lib crypt32.lib gdi32.lib user32.lib crypt32.lib

```

Comment:
Indeed, my typo. Too much of makefiles today :) I was basically refering to https://github.com/openssl/openssl/blob/master/NOTES.WIN#L114, but i see now the ws2_32 and advapi32 are added with WIN_LIBS, so we should be all fine with accordance to the doc.